penn/news.py

```diff
@@ -33,4 +34,4 @@ def search(self, keyword):
 
         data = self._request(ENDPOINTS['SEARCH'], params)
         data['result_data'] = filter(lambda x: type(x) == dict, data['result_data'])
-        return data
+        return list(data)
```

Comment:
Why does this return a list and https://github.com/pennlabs/penn-sdk-python/pull/45/files#diff-df24122eb903ed0acd46dfbe5fbfa076L35 doesn't?